src/redux/actions/search.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
```

Comment:
`Recommend-Must`
캐시기능을 사용하기 위해 './utilites/axiosCached.js' 로 변경해주시면 좋을 것 같습니다.

```suggestion
import axios from './utilities/axiosCached';
```